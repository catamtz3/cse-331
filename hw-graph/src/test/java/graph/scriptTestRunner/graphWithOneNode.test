# Create a graph
CreateGraph singleNode

#Add node to the graph
addNode singleNode a

# Get the nodes of the graph
ListNodes singleNode