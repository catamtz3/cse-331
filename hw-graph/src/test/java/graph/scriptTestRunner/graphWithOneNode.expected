# Create a graph
Created graph singleNode

#Add node to the graph
Added node a to singleNode

# Get the nodes of the graph
singleNode contains: a